gitlab/resource_gitlab_service_microsoft_teams.go

```diff
@@ -0,0 +1,181 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabServiceMicrosoftTeams() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabServiceMicrosoftTeamsCreate,
+		Read:   resourceGitlabServiceMicrosoftTeamsRead,
+		Update: resourceGitlabServiceMicrosoftTeamsUpdate,
+		Delete: resourceGitlabServiceMicrosoftTeamsDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"title": {
```

Comment:
Can you double check your attributes one more time? `title` does not appear to be used, and it is not mentioned in the API docs. https://docs.gitlab.com/ee/api/integrations.html#createedit-microsoft-teams-integration

Also a reminder to double check that each attribute is listed on the new documentation page.

Reply:
Sorry, my mistake: The `title` is returned by the API, but apparently only used in the frontend (value is "Microsoft Teams notifications"). I'll remove it and check all other attriutes another time.